skactiveml/pool/_expected_error.py

```diff
@@ -91,6 +92,9 @@ def query(self, X_cand, X, y, batch_size=1, return_utilities=False,
                             "'ClassFrequencyEstimator'.")
         check_classifier_params(self.classes, self.missing_label, self.C)
 
+        # Check random state
+        self.random_state = check_random_state(self.random_state)
```

Comment:
`random_state` should be a local variable here (at least other than `self.random_state`)

See also other methods

Reply:
#50